usaspending_api/transactions/delta_models/summary_state_view.py

```diff
@@ -0,0 +1,65 @@
+SUMMARY_STATE_VIEW_COLUMNS = {
+    "duh": "STRING",
+    "action_date": "DATE",
+    "fiscal_year": "INTEGER",
+    "type": "STRING",
+    "distinct_awards": "ARRAY<STRING>",
+    "pop_country_code": "STRING",
+    "pop_state_code": "STRING",
+    "generated_pragmatic_obligation": "NUMERIC(23,2)",
+    "federal_action_obligation": "NUMERIC(23,2)",
+    "original_loan_subsidy_cost": "NUMERIC(23,2)",
+    "face_value_loan_guarantee": "NUMERIC(23,2)",
+    "counts": "INTEGER",
+}
+
+summary_state_view_create_sql_string = fr"""
+    CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
+        {", ".join([f'{key} {val}' for key, val in SUMMARY_STATE_VIEW_COLUMNS.items()])}
+    )
+    USING DELTA
+    LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
+"""
+
+summary_state_view_load_sql_string = rf"""
```

Comment:
I think remove this load_sql_string for now until it is getting used. As-is this SQL won't work because it looks like you copied over the JSON file's contents which means you have a bunch of `"` all throughout the SQL. And if this is going to be here it should be correct / tested otherwise we might see it and wrongly think a story was already implemented.